src/Uno.UWP/Data/Xml/Dom/XmlCDataSection.cs

```diff
@@ -25,38 +25,38 @@ public string Data
 		public object Prefix
 		{
 			get => _backingDataSection.Prefix;
-			set => _backingDataSection.Prefix = value?.ToString();
+			set => _backingDataSection.Prefix = value.ToString()!;
```

Comment:
What happens when setting null to Prefix on WinRT?

Reply:
It seems setting to null is accepted on WinUI. Tested with the following snippet (not really a CData but I'd assume all node types will behave the same, `Prefix` is an interface member anyways and the annotation of all implementers should be the same of the interface):

```csharp
        var doc = new XmlDocument();
        doc.LoadXml(@"
<p1:Hello xmlns:p1=""https://platform.uno/prefix1"" xmlns:p2=""https://platform.uno/prefix2"">
    <Child>ChildContent</Child>
</p1:Hello>
");

        doc.FirstChild.Prefix = null;
```

I was following the same nullability annotation from `System.Xml`. But looks like System.Xml actually allows for null even though it's not annotated nullable.

I'll adjust but will have to have suppressions to account for the wrong System.Xml annotations.

Reply:
btw, our current behavior (relying on System.Xml) doesn't match WinUI.

On WinUI, if you set the prefix to null and try to get the value back, it returns null (same as you assigned).

In System.Xml (and hence Uno), if you set the prefix to null and try to get the value back, it returns empty string.